build.rs

```diff
@@ -167,10 +147,22 @@ fn ignore(testsuite: &str, testname: &str, strategy: &str) -> bool {
             (_, _) if testname.starts_with("simd") => return true,
             (_, _) if testsuite.ends_with("multi_value") => return true,
             // Lightbeam doesn't support float arguments on the stack.
-            ("spec_testsuite", "call.wast") => return true,
+            ("spec_testsuite", "call") => return true,
             _ => (),
         },
         "Cranelift" => match (testsuite, testname) {
+            ("simd", "simd_bit_shift") => return true, // FIXME Unsupported feature: proposed SIMD operator I8x16Shl
+            ("simd", "simd_const") => return true, // FIXME Invalid input WebAssembly code at offset 474: Unexpected data at the end of the section
```

Comment:
@alexcrichton, I haven't investigated this yet but his might be a `wast` issue

Reply:
I think this is an issue because wasmparser needs an update since the binary encoding for these sorts of table elements has changed, but I have yet to dig in deeper to confirm that this is what's needed.